src/kaltura-player.ts

```diff
@@ -257,6 +257,9 @@ export class KalturaPlayer extends FakeEventTarget {
     this._playlistManager.load(playlistData, playlistConfig, entryList);
   }
 
+  public setLocalPlayerReset(reset: boolean) {
+    this._localPlayer._reset = reset;
```

Comment:
expose an api on playkit player to do this

something like

clearReset() {
this._reset = false;
}